doc/installation.rst

```diff
@@ -264,6 +265,19 @@ In summary:
 
    This installs all of Spyder's dependencies into the environment along with the stable/packaged version of Spyder, and then removes Spyder itself.
 
+   .. note::
+
+      Following the separation of ``spyder-kernels`` into its own modular package in Spyder 3.3.0, you'll need to have the corresponding version of that package available—``0.x`` for Spyder 3 (``3.x`` branch), and ``1.x`` for Spyder 4 (``master`` branch).
```

Comment:
I think this is better:

> Following the separation of the code that powers Spyder's consoles into its own modular package (called ``spyder-kernels``) ...

Reply:
Thanks; I back-ported the same revised line you requested on the Contributing Guide in spyder-ide/spyder#7418 .